app/models/offer.rb

```diff
@@ -111,11 +111,8 @@ def send_update
     user = Api::V1::UserSerializer.new(User.current_user)
     object = Api::V1::OfferSerializer.new(self,
       { exclude: Offer.reflections.keys.map(&:to_sym) })
-    PushService.new(
-      channel: "user_#{self.created_by_id}",
-      event: 'update_store',
-      data: { item: object, sender: user, operation: :update }
-    ).notify
+    channel = "user_#{self.created_by_id}"
+    PushService.new.send_update_store(channel, {item: object, sender: user, operation: :update}, "offer#{id}")
```

Comment:
Line is too long. [110/80]<br>Space inside { missing.<br>Space inside } missing.